example/apache_echarts.html

```diff
@@ -0,0 +1,2401 @@
+    "player146": {
+      "id": "player146",
+      "labels": [
+        "player"
+      ],
+      "props": {
+        "age": "33",
+        "id": "player146",
+        "name": "Dwight Howard"
+      }
+    },
+    "team200": {
+      "id": "team200",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team200",
+        "name": "Warriors"
+      }
+    },
+    "team203": {
+      "id": "team203",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team203",
+        "name": "Trail Blazers"
+      }
+    },
+    "team204": {
+      "id": "team204",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team204",
+        "name": "Spurs"
+      }
+    },
+    "team206": {
+      "id": "team206",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team206",
+        "name": "Jazz"
+      }
+    },
+    "team208": {
+      "id": "team208",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team208",
+        "name": "Kings"
+      }
+    },
+    "team214": {
+      "id": "team214",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team214",
+        "name": "Suns"
+      }
+    },
+    "team215": {
+      "id": "team215",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team215",
+        "name": "Hornets"
+      }
+    },
+    "team218": {
+      "id": "team218",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team218",
+        "name": "Raptors"
+      }
+    },
+    "team219": {
+      "id": "team219",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team219",
+        "name": "76ers"
+      }
+    },
+    "team221": {
+      "id": "team221",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team221",
+        "name": "Bulls"
+      }
+    },
+    "team222": {
+      "id": "team222",
+      "labels": [
+        "team"
+      ],
+      "props": {
+        "id": "team222",
+        "name": "Hawks"
+      }
+    }
+  }
+}
+
+myChart.hideLoading();
+
+// Directly using the provided data to build the graph
+let graph = {
+  nodes: data.nodes.map(node => ({
+    ...node,
+    symbolSize: 5,
+    name: node.props.name ? node.props.name : node.id,
+    value: node.props,
+    category: node.tags && node.tags.length > 0 ? node.tags[0] : undefined
+  })),
+  links: data.edges.map(edge => ({
+    source: edge.src,
+    target: edge.dst,
+    name: edge.name,
+    value: edge.props
+  }))
+};
+
+// handle categories(node label)
+let categories = [];
+graph.nodes.forEach(function (node) {
+  if (node.tags && node.tags.length > 0) {
+    node.tags.forEach(function (tag) {
+      if (categories.indexOf(tag) === -1) {
+        categories.push(tag);
+      }
+    });
+  }
+});
+graph.categories = categories.map(function (category) {
+  return {name: category};
+});
+
+option = {
```

Comment:
It seems that the G6's APIs are more suitable for Graph special visualization?

https://github.com/antvis/G6

Reply:
Make sense, I can provide mainstream libs flavor of output, including g6.